command/resource/client/grpc-flags.go

```diff
@@ -17,6 +17,12 @@ type GRPCFlags struct {
 	caPath    TValue[string]
 	token     TValue[string]
 	tokenFile TValue[string]
+
+	// flags that are not merged into GRPCConfig
+	namespace TValue[string]
+	partition TValue[string]
+	peername  TValue[string]
+	stale     TValue[bool]
```

Comment:
Why are these flags here if they're not related to grpc specific flags?

Reply:
Thanks for making me reflecting upon the motivation behind the flags.
I've created another new file to host these flags, please review.
Thanks again!